### PR TITLE
SEP Data route

### PR DESCRIPTION
**Pull request by MikeyManoguerra**
creates an endpoint to post SEP data to.
#384

### CODE REVIEW

File: core/tests/sep_data.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            "needles_in": 5,
+            "needles_out": 5,
+        }
+
+        response = self.client.post(url, data, format="json", follow=True, **headers)
+        content = json.loads(response.content)
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertTrue(Sep_Data.objects.filter(pk=content["id"]).exists())
+
```
Comment (JackRyan1989) [inline core/tests/sep_data.py:72]: Not sure this is necessary, but maybe add a test to show that we cannot post using uds_provider? That would just be a complete test of the permissions set up.

File: core/queue/serializers.py [hunk 2]
```diff
@@ -1,9 +0,0 @@
-from rest_framework import serializers
```
Comment (JackRyan1989) [inline core/queue/serializers.py:1]: Why was this file deleted?

**Review by JackRyan1989 — COMMENTED** [review]
Looks really good! Just had a few simple questions.

File: core/queue/serializers.py (hunk 2 above)
Comment (MikeyManoguerra) [reply core/queue/serializers.py:1]: this serializer was never used.  The queue just uses the already existing Program serializer , since there is no queue table.

File: core/tests/sep_data.py (hunk 1 above)
Comment (MikeyManoguerra) [reply core/tests/sep_data.py:72]: I added one!

Comment (JackRyan1989) [reply core/tests/sep_data.py:72]: Perfecto!

**Review by JackRyan1989 — APPROVED** [review]
